software/fastqc/main.nf

```diff
@@ -26,6 +26,7 @@ process FASTQC {
     if (single_end) {
         """
         [ ! -f  ${name}.fastq.gz ] && ln -s $reads ${name}.fastq.gz
+        ls -la
```

Comment:
Is this on purpose or left over from debugging?

Reply:
Debugging 👀